Button Styles

justinacton opened the pull request:
Added button styles for each theme to apply theme styles to dnnPrimary,
dnnSecondary, and Bootstrap primary buttons

justinacton committed:
Button Styles

Added button styles for each theme to apply theme styles to dnnPrimary,
dnnSecondary, and Bootstrap primary buttons

dnnconsulting commented:
Hi Justin, sorry for the delay! I'll try and review you additions within the next couple of days. On first look it looks like a great addition to be included.

Thanks.
Geoff

justinacton commented:
No problem, Geoff.  I'm sure you're busy with plenty of other things.

Thanks,

Justin

On Tue, May 3, 2016 at 7:42 AM, Geoff Barlow notifications@github.com
wrote:

> Hi Justin, sorry for the delay! I'll try and review you additions within
> the next couple of days. On first look it looks like a great addition to be
> included.
> 
> Thanks.
> Geoff
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/dnnconsulting/DnnC.Bootster/pull/5#issuecomment-216515406

justinacton committed:
added additional menu type

added another menu option for a menu that has clickable parent nodes for
dropdown menus.  For example, the default menu doesn't allow you to
click and go to the parent page of the dropdown item, the added menu
type allows this.

Also added info in README file on how to change menu types and added
info for changing themes.
